task/Taskfile.lagoon.yml

```diff
@@ -0,0 +1,100 @@
+# This is a go-task file solely for Lagoon integration.
+# It can be used independently but is meant to be used
+# in combination with the go-task file in the root of the project.
+version: "3"
+
+vars:
+  # The name of the project on Lagoon.
+  LAGOON_PROJECT: "dpl-cms"
+
+tasks:
+  backup:restore:
+    desc: Restore the latest backup of files and database from a Lagoon project
+    cmds:
+      - task: backup:restore:database
+      - task: backup:restore:files
+
+  backup:restore:database:
+    desc: Restore the latest backup of the database from a Lagoon project
+    deps: [deps:lagoon, deps:jq, login]
+    vars:
+      LAGOON_ENVIRONMENT: main
+    cmds:
+      - |
+        BACKUPS=$(lagoon list backups -p ${LAGOON_PROJECT} -e {{.LAGOON_ENVIRONMENT}} --output-json);
+        BACKUP_ID=$(echo $BACKUPS | jq -r ".data[] | select(.source == \"mariadb\") | .backupid" | head -n 1);
+        BACKUP_URL="Error";
+        while [[ $BACKUP_URL == *"Error"* ]]; do
+          $(lagoon retrieve backup -p ${LAGOON_PROJECT} -e {{.LAGOON_ENVIRONMENT}} --backup-id ${BACKUP_ID} --force) 2>/dev/null || true;
+          BACKUP_URL=$(lagoon get backup -p ${LAGOON_PROJECT} -e {{.LAGOON_ENVIRONMENT}} --backup-id ${BACKUP_ID});
```

Comment:
`lagoon get backup` will retur an error message á la.`Error: Backup is not restored` if the backup is not yet available.

Reply:
So you made a workaround?